editor/common/src/EditorProps.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+/** @internal */
+export const editCommandApi = {
+  start: "imodeljs.editcmd.start",
+  call: "imodeljs.editcmd.call",
+};
+
+/** @internal */
+export type CommandError =
+  "CommandNotFound" |
+  "Exception" |
+  "MethodNotFound" |
+  "NoActiveCommand";
+
+/** @internal */
+export interface CommandResult<T> {

```

Comment:
Can a CommandResult have both an error and a result?

Reply:
I'd like to revisit this after you get the packages merged in. Seems like a CommandResult should be exactly one of either an error or a result.

Reply:
I dislike this too, I merely copied what we do on our native code in `ErrorStatusOrResult` in `NativeLibrary.ts` I doubt it will survive